upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,41 @@
 == Migration Changes
 
+=== Migrating to 13.0.0
+
+==== Upgrade to Wildfly 23
+
+The {project_name} server was upgraded to use Wildfly 23 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, but a few changes related to the migration, which are worth mentioning.
+
+Dependency updates::
+  The dependencies were updated to the versions used by Wildfly 23 server. For example, Infinispan is now `11.0.9.Final`.
+
+Configuration changes::
+  A few configuration changes exist in the `standalone(-ha).xml` and `domain.xml` files. You should follow the <<_install_new_version>>
+  section to handle the migration of configuration files automatically. If more detail is needed, because, for example, you did some
+  configuration changes on your own, the list of the most important changes follows:
+  * The `module` attribute of Infinispan cache containers is now *deprecated* (unused) and was *replaced with* the `modules` attribute,
```

Comment:
is *replaced with*

Reply:
Replaced both instances. Also replaced `were replaced` with `are replaced` and couple of other (related) ones. See latest version.